src/test/java/org/jenkins_cli/plugins/ifdtms/CucumberPostBuildTest.java

```diff
@@ -2,11 +2,12 @@
 
 import static org.junit.Assert.*;
 
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class CucumberPostBuildTest {
 
-	@Test
+	@Ignore
```

Comment:
The usual syntax is to annotate with both `@Ignore` and `@Test`:

```suggestion
	@Ignore
	@Test
```